cfpb/consumerfinance.gov
# Organize functional Cypress tests to run in groups on Jenkins

- Add a series of Cypress containers and config spec sets in docker-compose.e2e.yml
- Move cypress integration page tests into consumer-tools folder
- Disable screenshot capture when a test fails
- Install and try to use `docker-compose` instead of `docker run` in Jenkinsfile
- Fix admin cypress test

## How to test this PR

1. Verify that all cypress tests run in groups in docker containers
2. Check no groups of tests fail to complete due to timeout
3. Confirm that the admin test passes


## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
